src/components/AccountDrawer/MiniPortfolio/Tokens/index.tsx

```diff
@@ -22,8 +22,42 @@ import PortfolioRow, { PortfolioSkeleton, PortfolioTabWrapper } from '../Portfol
 
 const HIDE_SMALL_USD_BALANCES_THRESHOLD = 1
 
-function meetsThreshold(tokenBalance: TokenBalance, hideSmallBalances: boolean) {
-  return !hideSmallBalances || (tokenBalance.denominatedValue?.value ?? 0) > HIDE_SMALL_USD_BALANCES_THRESHOLD
+function meetsThreshold(tokenBalance: TokenBalance) {
+  const value = tokenBalance.denominatedValue?.value ?? 0
+  return value > HIDE_SMALL_USD_BALANCES_THRESHOLD
+}
+
+// exporting for use in tests only
+// eslint-disable-next-line import/no-unused-modules
+export function splitHiddenTokens(
```

Comment:
we generally should try to avoid exporting only for tests. An alternative would be to do a cypress test for the `MiniPortfolio/Token` component with mock input. See `src/components/AccountDrawer/MiniPortfolio/PortfolioLogo.test.tsx` for an example

Reply:
@cbachmeier i think starting from scratch i'd do that and mock the `useCachedPortfolioBalancesQuery` hook, but rewriting it now feels like it ends up testing the same basic thing but with a decent amount of added effort. sound fair?

i did refactor the `splitHiddenTokens` into its own file and moved it to utils which actually feels right given we likely want this function to be shared anywhere we want to hide tokens, which could be (is?) in more places than here. lmk if that sounds alright.